pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -157,7 +157,6 @@ iamcco/coc-vimlsp
 ianks/vim-tsx
 idanarye/vim-merginal
 idris-hackers/idris-vim
-ihsanturk/neuron.vim
```

Comment:
This is unrelated to this commit, right?

Reply:
That's the original plugin. I figured since it's no longer maintained, we could take it out and replace it with the new one. Not sure how to resolve the naming conflict if we should actually be keeping both.

Reply:
I totally agree with your change but you changed it in [`a79c400` (#101385)](https://github.com/NixOS/nixpkgs/pull/101385/commits/a79c4000a51f3b104e0aacd2b27a4cd03e2bd88b) which you named ``vimPlugins: resolve github repository redirects``. I think the change is wrong in this particular commit and maybe needs its own commit or fits into some other commit better.

Reply:
Gotcha. All the commits were created by the `update.py` script. I'll go through with a rebase and organize the changes a little better.